common/changes/@itwin/grouping-mapping-widget/gm-extension_2022-09-07-05-13.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "Rename to customUIProviders and default providers will be replaced if custom ones are provided",
```

Comment:
Why did you add a separate changelog entry for this? I don't think it's a rename if that functionality is being added.

Reply:
I still don't understand why didn't you simply change the existing changelog and added a new one? In a single release you add something and in the same release you already rename it. The rename does not require a changelog entry, because the end user will see it as a single Add, not Add + Rename.